specification/riskiq/Easm/tspconfig.yaml

```diff
@@ -1,6 +1,14 @@
 parameters:
   "service-dir":
     default: "sdk/easm"
+  "python-sdk-folder":
+    default: "{project-root}/azure-sdk-for-python/"
+  "java-sdk-folder": 
+    default: "{project-root}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{project-root}/azure-sdk-for-js/"
+  "csharp-sdk-folder":
+    default: "{project-root}/azure-sdk-for-net/"
```

Comment:
I don't think we need these xx-folder?
sample config https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml